automation_tools/__init__.py

```diff
@@ -1358,6 +1358,34 @@ def configure_realm(admin_password=None, keytab_url=None, realm=None,
     run('service foreman-proxy restart')
 
 
+def apply_hotfix():
+    """Apply a hotfix for the Satellite6 Server
+
+    Expects the following environment variables:
+
+    HOTFIX
+        Specifies the choice for the hotfix, the default or custom.
```

Comment:
So `Default` is when Hotfix is available through any repo and `custom` when its available through custom script ?